zerver/lib/test_classes.py

```diff
@@ -1129,15 +1307,28 @@ def common_subscribe_to_streams(
         invite_only: bool = False,
         is_web_public: bool = False,
         allow_fail: bool = False,
-        **kwargs: ClientArg,
+        skip_user_agent: bool = False,
+        follow: bool = False,
+        secure: bool = False,
+        intentionally_undocumented: bool = False,
```

Comment:
Are these used by any callers? Did a quick look through the calls right now and didn't notice any. If that's the case, then wouldn't it be cleaner to not have these arguments here at all?

Reply:
Same questions for a few of the other functions like this - `send_webhook_payload, api_stream_message,  check_webhook, send_and_test_private_message`.

Reply:
Oh, that makes sense. Thanks!